pkg/bootstrap/manifests/klusterlet/clusterrole_priority_class.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Add a todo/note that we will remove this after 2 releases?

Reply:
Updated.